download.go

```diff
@@ -15,29 +17,29 @@ func extractArchive(path string) error {
 	return command.Run([]string{"tar", "zxvf", path})
 }
 
-func download(url string, logName string) error {
-	args := []string{"wget", url}
-	if command.VerboseEnabled {
-		return command.Run(args)
+func downloadForBuilder(b *builder.Builder) error {
```

Comment:
I don't think this function name should be renamed. `download(b *builder.Builder)` is good.